External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -2318,12 +2331,33 @@ static public void HandleAllClassesCompletion(ScintillaControl Sci, string tail,
                     known = found;
                 }
             }
-
+            
             if (ASContext.Context.Features.hasDelegates && !ASContext.Context.CurrentClass.IsVoid())
```

Comment:
Right, I didn't do it just to mark a distinction between both things. But I'll do it.